inst/tests/tests.Rraw

```diff
@@ -9789,6 +9788,11 @@ indices(DT, vectors = TRUE)
 test(1749.1, indices(DT), c("A__B","A","B"))
 test(1749.2, indices(DT, vectors = TRUE), list(c("A","B"),"A","B"))
 
+# allow incomplete specification in setcolorder, #592
+DT = data.table(a = 1:3, b = 2:4, c = 3:5)
+test(1750.1, names(setcolorder(DT, "b")), c("b", "a", "c"))
+test(1750.2, names(setcolorder(DT, 3)), c("c", "b", "a"))
```

Comment:
yeah, but your chunk changes behavior of the next chunk, `setdiff` will both remove NAs and duplicates so later validation will not raise error when it should.